src/core/drive/form_submission.ts

```diff
@@ -56,6 +70,20 @@ export class FormSubmission {
     return expandURL(this.action)
   }
 
+  get body() {
+    if (this.enctype == FormEnctype.urlEncoded || this.method == FetchMethod.get) {
+      return new URLSearchParams(this.formData as any)
+    } else {
+      return this.formData
+    }
+  }
+
+  get enctype(): FormEnctype {
+    const enctype = this.submitter?.getAttribute("formenctype") || this.formElement.enctype
+
+    return formEnctypeFromString(enctype) || FormEnctype.urlEncoded
```

Comment:
```suggestion
    return formEnctypeFromString(enctype)
```

The `||` is never hit because of the default function return.